<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image: url(:/assets/background.png);
    background-repeat: no-repeat;
    background-position: center;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="leftMenu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>1081</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
	background-color: rgba(255, 255, 255, 0.2);
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginPage">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>330</y>
        <width>741</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    background-color: rgba(255, 255, 255, 0.2);
    border: 1px solid white;
    border-radius: 15px;
}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="usernameLine">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>140</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid white;
    border-radius: 10px;
    padding: 5px;
    color: white;
}
</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLine">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>180</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid white;
    border-radius: 10px;
    padding: 5px;
    color: white;
}
</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Remember me</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>303</x>
         <y>275</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    border-radius: 20px;
    padding: 10px 20px;
    color: black;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #e0e0e0;
}
</string>
       </property>
       <property name="text">
        <string>login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>221</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Arial'; font-size:48pt; font-weight:700;&quot;&gt;LOGIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>330</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Don't have an account yet ?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="registerButton">
       <property name="geometry">
        <rect>
         <x>399</x>
         <y>326</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>349</x>
         <y>229</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Forgot password ?</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
